examples/simple_neo4j.py

```diff
@@ -0,0 +1,67 @@
+import sycamore
+from sycamore.transforms.extract_document_structure import StructureByDocument
+from sycamore.transforms.extract_graph_entities import EntityExtractor
+from sycamore.transforms.extract_graph_relationships import RelationshipExtractor
+from sycamore.transforms.partition import ArynPartitioner
+
+from sycamore.llms import OpenAI, OpenAIModels
+
+from typing import Optional
+from pydantic import BaseModel, Field
+
+llm = OpenAI(OpenAIModels.GPT_4O_STRUCTURED)
+ctx = sycamore.init()
+
+
+class Report(BaseModel):
+    ID: str = Field(
+        description="This ID can be found under either Accident Number, Incident Number, or Occurance Number"
+    )
+
+
+class Aircraft(BaseModel):
+    registration: str
+    make: str
+    model: str
```

Comment:
should all these guys have descriptions too?

Reply:
They don't have to, but they can. It just helps the LLM generate context for what to extract, usually I only add them for stuff that is unclear(i.e. the report id since it could be found under several different fields)